deployment/bare_metal/.env.dist

```diff
@@ -30,6 +30,11 @@ TYPEORM_LOGGING_RELATIVE_PATH=../deployment/bare_metal/log/typeorm.backend.log
 
 WEBHOOK_ELOPAGE_SECRET=secret
 
+COMMUNITY_NAME=Gradido Development Stage1
+COMMUNITY_URL=https://stage1.gradido.net/
+COMMUNITY_REGISTER_URL=https://stage1.gradido.net/register
+COMMUNITY_DESCRIPTION=Gradido Development Stage1 Test Community
```

Comment:
So we use this bare_metal to deploy, and each server get's his own repo? Or how should this work?

In my eyes if we use this on stage2 it would not work or do I missunderstand something?

Reply:
you dcheckout the repo and put an .env in this folder. The stage1 is just as an example so its not so cryptic.